py-scripts/lf_interop_ping_plotter.py

```diff
@@ -124,8 +151,30 @@ def __init__(self,
                  do_webUI=False,
                  ui_report_dir=None,
                  debug=False,
+                 file_name=None,
+                 profile_name=None,
+                 group_name=None,
+                 eap_method=None,
+                 eap_identity=None,
+                 ieee80211=None,
```

Comment:
Similar comments in other PRs, this should be 8021x. Since this is first round of review for this, let's fix this before merge.